README.md

```diff
@@ -65,6 +65,39 @@ customElements.define('new-element', NewElement);
 
 See examples in the demo or try this live [JS bin](https://jsbin.com/zitotejimi/edit?html,output).
 
+## Uprading from 3.x
+
+The component has been migrated to LitElement and uses TypeScript now. This migration introduced two breaking changes.
+
+### Removed Polymer-specific `selection-changed` event used for 2-way bindings
+
+There were two events that notify about chart selection changes: `google-chart-select` and the Polymer-generated `selection-changed`.
+For consistency with other events (e.g. `google-chart-ready`), we keep only `google-chart-select`.
+
+Polymer components using 2-way binding for the selection should be updated to explicitly name the event - https://polymer-library.polymer-project.org/3.0/docs/devguide/data-binding#two-way-native. Note the `::google-chart-select` part.
```

Comment:
"Polymer components using this feature must be updated to explicitly name the selection event ([details](https://polymer-library.polymer-project.org/3.0/docs/devguide/data-binding#two-way-native)). In the example below, note the addition of `::google-chart-select`."